lib/systems/examples.nix

```diff
@@ -63,6 +63,18 @@ rec {
     platform = platforms.fuloong2f_n32;
   };
 
+  muslpi = (removeAttrs raspberryPi [ "libc" ]) // {
```

Comment:
Err I meant literally remove `libc` from above. It should be inferred just fine!

Reply:
Haha, sure thing.  I agree that's nicer/cleaner, I just already feel I touch "unrelated" things in this PR and leaving existing examples as-is seemed easiest way to make it easy for reviewers to see I don't break them :).

Anyway, removed!